### DISCUSSION TITLE
今天玩使命召唤262四枪没打死人被反杀，望周知

### DISCUSSION THREAD

**Discussion by love262-wkx** [share121/inter-knot#5073] [其他]
![img1030322001_1726463682790](https://github.com/user-attachments/assets/993df7a7-5381-47c8-b854-1ee8ebc5d12a)

**Comment by Yizakl** [comment]
666Co玩家

**Comment by Sorasakirin** [comment]
不玩co

**Comment by koshsa** [comment]
262是这样的，不过我是k狙
